app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
@Flaburgan : Well it actually was like that:

    %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}

But the apple-touch-icon.png is also used by Firefox and Firefox mobile as the favicon which makes it look messy because this image is too big to make a proper icon. Unless I missed something. I changed a lot of things here, I might have made mistakes.